README.md

```diff
@@ -256,11 +249,11 @@ Unzip the file. Following is the directory structure for Linux 64-bit zip. The s
 For compilation, you need to include the `hazelcast/include` directory in your distribution. You also need to link your application to the appropriate static or shared Hazelcast library. e.g. cmake lines needed to include and link to Hazelcast 64-bit library:
 ```Cmake
 	include_directories(hazelcast/include)
-    link_directories(hazelcast/Linux_64/lib)
+    link_directories(hazelcast/lib)
 	link_libraries(HazelcastClient4.0_64)
 ``` 
  
-If you want to use the TLS feature, use the `lib` directory with TLS support enabled, e.g., `hazelcast/Linux_64/lib/tls`. If you are using TLS, then you also need to link to OpenSSL and link to the OpenSSL libraries. E.g. if you are using cmake, we do the following:
+If you want to use the TLS feature, use the `lib` directory with TLS support enabled, e.g., `hazelcast/lib/tls`. If you are using TLS, then you also need to link to OpenSSL and link to the OpenSSL libraries. E.g. if you are using cmake, we do the following:
```

Comment:
... need to link to OpenSSL and its libraries. For example, if you are using  ...